profitbricks.go

```diff
@@ -50,12 +57,15 @@ func RestyClient(username, password, token string) *Client {
 	})
 	c.AddRetryCondition(
 		func(r *resty.Response, err error) bool {
+			// Only retry on read-only requests
```

Comment:
I think this comment is superfluous now that you wrapped the check in a function